prisma/migrations/20260725101500_harden_template_tag_arrays/migration.sql

```
@@ -0,0 +1,8 @@
+-- Enforce the required-array invariant at the database level for the template
+-- tag arrays. Both `tags` and `filterableTags` are non-null at the Prisma/app
+-- layer (Prisma scalar lists are always non-null), but the underlying columns
+-- were created nullable, so a non-Prisma write could insert NULL and violate the
+-- contract that dataset reads rely on. Backfill any stray NULLs, then constrain.
+UPDATE "templates" SET "tags" = ARRAY[]::TEXT[] WHERE "tags" IS NULL;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)